static/app/components/modals/inviteMembersModal/useInviteModal.tsx

```diff
@@ -131,11 +133,13 @@ export default function useInviteModal({organization, initialData, source}: Prop
               ? errorResponse.email[0]
               : errorResponse.email;
 
-        const error = emailError || t('Could not invite user');
+        const orgLevelError = errorResponse?.organization || null;
```

Comment:
is `errorResponse?.organization` the organization?

Reply:
You would think right? hahah
It's actually the key in the object for the error we respond with https://github.com/getsentry/sentry/blob/bdb19d5895948d63c68c862a0feae0a1e529759c/src/sentry/api/endpoints/organization_member/index.py#L236

Reply:
Oh weird, that's poorly named.

Reply:
I think the intention was that this is an error at the organization level? But agreed, we could improve the wording or even the format here